packages/browser/package.json

```diff
@@ -95,5 +71,8 @@
   },
   "publishConfig": {
     "access": "public"
+  },
+  "devDependencies": {
+    "jest": "^26.4.2"
```

Comment:
Dependencies that ship their own binaries, like Jest, will still need to be available in the relevant packages so they can be run from there.